src/mpid/ch3/channels/nemesis/include/mpid_nem_defs.h

```diff
@@ -10,7 +10,7 @@
 #include "mpid_nem_datatypes.h"
 #include "mpi.h"
 #include "mpiimpl.h"
-#include "mpidu_shm.h"
+#include "mpidu_init_shm.h"
 
```

Comment:
Can you please combine the three commits into a single commit? They all look like same functionality to me.